components/experimental/src/dimension/provider/units.rs

```diff
@@ -12,6 +12,16 @@
 use icu_provider::prelude::*;
 use zerovec::ZeroMap;
 
+#[cfg(feature = "compiled_data")]
+/// Baked data
+///
+/// <div class="stab unstable">
+/// 🚧 This code is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. In particular, the `DataProvider` implementations are only
+/// guaranteed to match with this version's `*_unstable` providers. Use with caution.
+/// </div>
+pub use crate::provider::Baked;
```

Comment:
you shouldn't need this, you can always refer to the provider as `crate::provider::Baked`

Reply:
done